front/app/components/IdeasMap/index.tsx

```diff
@@ -179,296 +93,427 @@ const StyledIdeaMapCard = styled(IdeaMapCard)<{ isClickable: boolean }>`
-    if (!initiallySelectedMarkerId || initialMapCenter) return;
-    const point = selectedIdeaMarker?.attributes.location_point_geojson;
-
-    if (!point) {
-      // For whatever reason, ideaMarkers also includes ideas without
-      // markers. If the search params contain one of those,
-      // we do nothing
-      setInitiallySelectedMarkerId(null);
-      return;
-    }
-
-    const { coordinates } = point;
-    setInitialMapCenter([coordinates[1], coordinates[0]]);
-  }, [initiallySelectedMarkerId, initialMapCenter, selectedIdeaMarker]);
-
-  if (initiallySelectedMarkerId && !initialMapCenter) {
-    return null;
-  }
+    const selectedIdeaData = ideasList?.find(({ id }) => id === selectedIdea);
 
-  return (
-    <Container
-      ref={containerRef}
-      className={className || ''}
-      id={id}
-      aria-labelledby={ariaLabelledBy}
-      tabIndex={tabIndex}
-    >
-      <InnerContainer
-        leftMargin={innerContainerLeftMargin}
-        isPostingEnabled={isIdeaPostingEnabled}
-      >
-        {isIdeaPostingEnabled && (
-          <InfoOverlay>
-            <InfoOverlayInner>
-              <InfoOverlayIcon name="info-outline" />
-              <InfoOverlayText>
-                {!isAdminOrModerator ? (
-                  <FormattedMessage
-                    {...(tablet
-                      ? messages.tapOnMapToAdd
-                      : messages.clickOnMapToAdd)}
-                  />
-                ) : (
-                  <FormattedMessage
-                    {...(tablet
-                      ? messages.tapOnMapToAddAdmin
-                      : messages.clickOnMapToAddAdmin)}
-                  />
-                )}
-              </InfoOverlayText>
-            </InfoOverlayInner>
-          </InfoOverlay>
-        )}
-
-        <ScreenReaderOnly>
-          <FormattedMessage {...messages.a11y_mapTitle} />
-        </ScreenReaderOnly>
-
-        {tablet && (
-          <CSSTransition
-            classNames="animation"
-            in={!!selectedIdeaMarker}
-            timeout={300}
+    return (
+      <>
+        <StyledMapContainer ref={containerRef}>
+          <InnerContainer
+            leftMargin={innerContainerLeftMargin}
+            isPostingEnabled={true}
           >
-            <StyledIdeaMapCard
-              ideaMarker={selectedIdeaMarker as IIdeaMarkerData}
-              onClose={deselectIdeaMarker}
-              isClickable={isCardClickable}
+            <EsriMap
+              initialData={{
+                center: mapConfig.data.attributes.center_geojson,
+                zoom: Number(mapConfig.data.attributes.zoom_level),
+                showLayerVisibilityControl: true,
+                showLegend: true,
+                zoomWidgetLocation: 'right',
+                onInit: onMapInit,
+              }}
+              height={isMobileOrSmaller ? '68vh' : '80vh'}
+              layers={
+                ideasLayer ? [...geoJsonLayers, ideasLayer] : geoJsonLayers
```

Comment:
This breaks memoization unfortunately, fixed it in this commit: https://github.com/CitizenLabDotCo/citizenlab/pull/7054/commits/fb2f21daf73998f2e6cb265ba8db81fc3f378283